mozilla-release/mobile/android/base/java/com/cliqz/AppearanceManager.java

```diff
@@ -23,9 +23,6 @@ public String getCurrent() {
     }
 
     public void change(String appearance) {
-        if (mCurrentApearance.equals(appearance)) {
```

Comment:
I would still keep this one. There is no need to call an action if there is no change